content/about/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: About
+type: page
+---
+
+## Introduction
+
+[BreadTube.tv](/) is the collaborative effort of Creators, Subscribers, Moderators, Developers, Designers, and Coordinators working together to list content that is posted on [r/BreadTube](/reddit) and related mediums.
+
+BreadTube is a reference to the left-wing anarchist Peter Kropotkin's 1982 book, ["The Conquest of Bread"](https://theanarchistlibrary.org/library/petr-kropotkin-the-conquest-of-bread). The goal of the community is to challenge the far-right content creators who have taken power on services like YouTube for the purpose of spreading hate and to instead educate people on how their world operates, the alternative possible visions for our future, and how we organize ourselves to get there.
+
+> ["The Making of a YouTube Radical" New York Times](https://www.nytimes.com/interactive/2019/06/08/technology/youtube-radical.html)
+
+This website is a resource for discovering and sharing BreadTube content, content is suggested by public users through various forms ([Channels](/channels/new), [Playlists](/playlists/new)), these submissions are re viewable by interested members on [Discord](httsp://breadtube.tv/discord) and then added to the source code by a team of volunteer developers.
```

Comment:
> `are re viewable by interested members on`

reviewable? Think there is a cheeky space in there.

I would change this to: `can be reviewed by any interested community members on...and later added to the source code...`, it's a small change but I think it puts greater emphasis on the community.